questionary/prompts/common.py

```diff
@@ -10,21 +10,30 @@
     Window,
 )
 from prompt_toolkit.validation import Validator, ValidationError
-from typing import Optional, Any, List, Text, Dict, Union, Callable, Tuple
+from typing import Optional, Any, List, Text, Dict, Union, Callable, Tuple, Callable
 
 from questionary.constants import (
     SELECTED_POINTER,
     INDICATOR_SELECTED,
     INDICATOR_UNSELECTED,
 )
 
+# This is a cut-down version of `prompt_toolkit.formatted_text.AnyFormattedText`
+# which does not exist in v2 of prompt_toolkit
+FormattedText = Union[
+    Text,
+    List[Tuple[Text, Text]],
+    List[Tuple[Text, Text, Callable[[Any], None]]],
+    None,
+]
```

Comment:
We are using prompt_toolkit v3, so I think you can use `prompt_toolkit.formatted_text.AnyFormattedText`?

Reply:
Questionary supports both versions from what I can tell.

Reply:
Ah yes you're right, my bad. Will let Tom review too, but looks good - thanks for the PR.